2019-12-02 14:34:59,434 manager INFO 1.1.0
2019-12-02 14:34:59,434 manager INFO Copyright (C) 2003-2010
2019-12-02 14:34:59,434 manager INFO   Noriaki Ando
2019-12-02 14:34:59,436 manager INFO   Intelligent Systems Research Institute, AIST
2019-12-02 14:34:59,436 manager INFO Manager starting.
2019-12-02 14:34:59,436 manager INFO Starting local logging.
2019-12-02 14:34:59,440 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-12-02 14:35:01,457 ManagerServant INFO Master manager not found
2019-12-02 14:35:01,457 manager WARNING Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2019-12-02 14:35:01,503 LocalServiceAdmin INFO All the local services are enabled.
2019-12-02 14:35:01,535 ec_base INFO setRate(1000000.000000) done
2019-12-02 14:35:01,535 ec_base INFO setRate(1000.000000) done
2019-12-02 14:35:01,535 rtobject INFO 1 execution context was created.
2019-12-02 14:35:01,535 rtobject INFO Initial active configuration set is default.
2019-12-02 14:35:03,229 SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-12-02 14:35:03,229 SdoServiceAdmin ERROR type  already exists.
2019-12-02 14:35:03,230 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2019-12-02 14:35:03,230 SdoServiceAdmin ERROR type 46823ea2-a4ae-4e5e-8ec5-21a6849f09ce already exists.
